CODEOWNERS

```diff
@@ -10,5 +10,11 @@
 # review when someone opens a pull request.
 *                                @google/trillian-team
 
-/*.proto                         @Martin2112 @AlCutter @pphaneuf
-/storage/storagepb/storage.proto @Martin2112 @AlCutter @pphaneuf
+/*.proto                         @mhutchinson @AlCutter @pphaneuf
+/storage/storagepb/storage.proto @mhutchinson @AlCutter @pphaneuf
+
+# Mitigation for https://github.com/google/trillian/issues/1297
+# Folks to watch out for hanges to MySQL schema and ensure that
+# there's a note added in a sensible location about how to
+# upgrade schema instances.
+/storage/mysql/schema/*          @mhutchinson @AlCutter @pphaneuf
```

Comment:
Maybe steal the cloudspanner sdl from my PR too and then we'll go with this one.